File: src/JavaScript/easy/string/problem2828.js
/**
 * @param {string[]} words
 * @param {string} s
 * @return {boolean}
 */
var isAcronym = function(words, s) {
    let wordsFirstLetter = []
    words.forEach(word => {
        wordsFirstLetter.push(word[0])
    })
    if(JSON.stringify(s.split('')) == JSON.stringify(wordsFirstLetter)) {
        return true
    } else {
        return false
    }
};

// let words = ["alice","bob","charlie"], s = "abc"
// let words = ["an","apple"], s = "a"
let words = ["never","gonna","give","up","on","you"], s = "ngguoy"

isAcronym(words, s)

File: src/Python/easy/string/problem2828.py
from typing import List

# First solution

class Solution:
    def isAcronym(self, words: List[str], s: str) -> bool:
        wordsFirstLetter = []
        for word in words:
            wordsFirstLetter.append(word[0])
        
        if(str(wordsFirstLetter) == str(list(s))):
            return True
        else:
            return False
        
    
# Second solution

class Solution:
    def isAcronym(self, words: List[str], s: str) -> bool:
        wordsFirstLetter = ''
        for word in words:
            wordsFirstLetter += word[0]
        
        if wordsFirstLetter == s:
            return True
        else:
            return False
        
words = ["alice","bob","charlie"]
s = "abc"


solution = Solution()
solution.isAcronym(words=words, s=s)